Client/Frontend/Browser/ClipboardBarDisplayHandler.swift

```diff
@@ -7,17 +7,22 @@ public struct ClipboardBarToastUX {
 }
 
 protocol ClipboardBarDisplayHandlerDelegate: class {
-    func shouldDisplayClipboardBar(absoluteString: String)
+    func shouldDisplayClipboardBar(_ clipboardBar: ButtonToast)
```

Comment:
We can prolly make this a bit swifty 3.0  `shouldDisplay(clipboardBar bar: ButtonToast)`